Bottom: 7a3d1a9dd90ee9a1812cc1193760379422358407
Top:    a08e539e76bc44c571aa1e354db82dd36a5a72ac
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-07 11:34:30 -0500

Refresh of add-docker-option

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 0727917d..79bbd612 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -58,6 +58,9 @@
     role: docker-openssl
     tags:
     - openssl_container
+  - name: Build and start Nexus Artifact Repository container
+    role: nexus3
+    tags: [nexus, container]
   - name: Build and start the SVN Docker container
     role: docker-svn
     svn_repos: 'ddio NCD_Release'
